contracts/src/deployers/ezeth/EzETHHyperdriveDeployerCoordinator.sol

```diff
@@ -0,0 +1,133 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.20;
+
+import { IERC20 } from "../../interfaces/IERC20.sol";
+import { IHyperdrive } from "../../interfaces/IHyperdrive.sol";
+import { IHyperdriveDeployerCoordinator } from "../../interfaces/IHyperdriveDeployerCoordinator.sol";
+import { IRestakeManager } from "../../interfaces/IRestakeManager.sol";
+import { FixedPointMath, ONE } from "../../libraries/FixedPointMath.sol";
+import { HyperdriveDeployerCoordinator } from "../HyperdriveDeployerCoordinator.sol";
+
+/// @author DELV
+/// @title EzETHHyperdriveDeployerCoordinator
+/// @custom:disclaimer The language used in this code is for coding convenience
+/// @notice The deployer coordinator for the EzETHHyperdrive implementation.
+///                    only, and is not intended to, and does not, have any
+///                    particular legal or regulatory significance.
+contract EzETHHyperdriveDeployerCoordinator is HyperdriveDeployerCoordinator {
+    using FixedPointMath for uint256;
+
+    /// @notice The Renzo contract.
+    IRestakeManager public immutable restakeManager;
+    /// @notice The ezETH token contract.
```

Comment:
Nit: This repo's style is to always put a newline between separate blocks of code. Looks like this change will need to be made in a bunch of places.

```suggestion

    /// @notice The ezETH token contract.
```

Reply:
I did a pass and I think I got all these.  I finished with the nits and now I'll start working on the updated share price and I'll do another pass to double check this.